src/main/scala/com/lucidworks/spark/SolrRelation.scala

```diff
@@ -146,9 +146,18 @@ class SolrRelation(
 
     try {
       val querySchema = if (!fields.isEmpty) SolrRelationUtil.deriveQuerySchema(fields, baseSchema) else schema
-      val docs = solrRDD.query(query)
-      val rows = SolrRelationUtil.toRows(querySchema, docs)
-      rows
+      if (!solrRDD.exportHandler.getOrElse(false) && !conf.useCursorMarks.getOrElse(false)) {
+        // Determine whether to use Streaming API (/export handler) when 'use_export_handler' option is not set
+        val useStreamingAPI: Boolean = SolrRelation.isStreamingPossible(querySchema, baseSchema, query)
+        val docs = solrRDD.useExportHandler(useStreamingAPI).query(query)
```

Comment:
Let's add an INFO log message here telling the user the outcome of the isStreamingPossible check